files/es/web/api/batterymanager/level/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: BatteryManager.level
+slug: Web/API/BatteryManager/level
+tags:
+  - API
+  - Battery API
+  - NeedsMarkupWork
+  - Property
+  - Reference
+translation_of: Web/API/BatteryManager/level
+browser-compat: api.BatteryManager.level
+---
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Indica el valor actual del nivel de carga, estos valores están entre `0.0` a `1.0`.
+
+## Sintaxis
+
+```js
+var level = battery.level;
+```
+
+`Level` es un número que representa el nivel de carga de la batería del sistema en una escala entre 0.0 y 1.0.
+Un valor de 0 significa que la `battery`, que es un objeto de tipo {{domxref("BatteryManager")}},
+está vacía y el sistema esta apunto de entrar en suspensión.
+Un valor de 1.0 significa que la `battery` está cargada al máximo.
+El valor 1.0 también nos indica que el sistema no puede determinar el nivel de carga de la batería o si no está alimentado por batería.
```

Comment:
Feel free to modify the suggestion slightly — new paragraphs should have one new line separating them, or just put all the text here into one big paragraph as follows (like the en-US page has it):

```suggestion
`Level` es un número que representa el nivel de carga de la batería del sistema en una escala entre 0.0 y 1.0. Un valor de 0 significa que la `battery`, que es un objeto de tipo {{domxref("BatteryManager")}}, está vacía y el sistema esta apunto de entrar en suspensión. Un valor de 1.0 significa que la `battery` está cargada al máximo. El valor 1.0 también nos indica que el sistema no puede determinar el nivel de carga de la batería o si no está alimentado por batería.
```

Reply:
If this isn't accepted, the text on lines 24 and 25 looks like it should be on the same line (currently the sentence is broken over two lines)

Reply:
it will be accepted, you have better Spanish than mine